docs/api/frameless-window.md

```diff
@@ -29,6 +29,8 @@ You can do so by specifying the `titleBarStyle` option:
 #### `hidden`
 
 Results in a hidden title bar and a full size content window, yet the title bar still has the standard window controls (“traffic lights”) in the top left.
+Additionally, if `enableBlinkFeatures: 'WebAppWindowControlsOverlay'` is set, the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and
```

Comment:
Can you update the document to clarify that, the Window Controls Overlay will be available when `win.setWindowButtonVisibility(true)` is called or when `titleBarStyle` is set to a value other than `default`?